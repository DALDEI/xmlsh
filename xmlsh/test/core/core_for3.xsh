function restart ()
{
    echo list
    for group ; do 
         for x in a b c  ; do
           echo $group $x  
       done
    done 
}

restart 1 2 3

# Test variant with NL after ;

set a b c
for x ; 
do 
echo $x
done

for x 
do 
echo $x
done

for x ; do 
echo $x
done
set a1 a2 a3 
for x ; do echo $x ; done

for x
do echo $x ; done
