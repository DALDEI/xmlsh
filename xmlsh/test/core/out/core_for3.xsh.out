list
1 a
1 b
1 c
2 a
2 b
2 c
3 a
3 b
3 c
a
b
c
a
b
c
a
b
c
a1
a2
a3
a1
a2
a3
